src/main/java/com/bytecoders/pharmaid/repository/PrescriptionRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.bytecoders.pharmaid.repository;
+
+import com.bytecoders.pharmaid.repository.model.Prescription;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+/**
+ * JPA prescriptions repository.
+ */
+public interface PrescriptionRepository extends JpaRepository<Prescription, String> {
+
+  Optional<Prescription> findByUserId(String userId);
```

Comment:
correct to assume this is for future use?

Reply:
Yes, thinking was that if we need to find all of a user's prescriptions, we would find the user first, then get their prescriptions.  Based it off of what I was reading with the `findByEmail()` function for `loginUser()`. 

Let me know if that is the right approach

Reply:
looks great